### ISSUE TITLE
ERROR OneTimeFeeder ... does not support required transactional operations!

### ISSUE THREAD

**Issue by johnjohndoe** [52North/sos-importer#37]
I successfully created a config file for a .csv file using the *52n-sos-importer-core* module (d678a8b6cabb1b43a6c2ee2a8a0c9db6e2a87bb4). I have an instance of [SOS server v.4.3.0](https://github.com/52North/SOS/releases/tag/4.3.0) running on *Tomcat 7* to store the data. I temporarily enabled transactional operations as shown in the screenshot:

![sos-transactions](https://cloud.githubusercontent.com/assets/144518/8289458/d597b5a2-191c-11e5-9dc8-136fb792dfb6.png)

I use the *52n-sos-importer-feeder* module to run the import. However it fails with the error:

> ERROR OneTimeFeeder ... does not support required transactional operations!

Here is the console output:

```
$ java -jar ./52n-sos-importer-feeder/target/52n-sos-importer-feeder-bin.jar 
    -c ~/Desktop/sensordata.config.xml -d ~/Desktop/sensordata.csv
20:19:01.509 [main] INFO  Feeder - Application started
		Implementation-Title: Feeding Module
		Implementation-Version: 0.4.0-SNAPSHOT
		Archiver-Version: Plexus Archiver
		Built-By: john
		Implementation-Vendor-Id: org.n52.sensorweb
		geotools-version: 9.5
		n52-oxf-version: 2.0.0-alpha.4-SNAPSHOT
		Manifest-Version: 1.0
		Implementation-Vendor: 52°North
		Main-Class: org.n52.sos.importer.feeder.Feeder
		build-time: 2015-04-13_21-27
		Created-By: Apache Maven
		Build-Jdk: 1.8.0_40
		n52-sos-importer-xmlbeans-version: 0.2
		HeapSize Information: max: 1303MB; total now: 88MB; free now: 76MB; used now: 11MB
		os.name: Linux; os.arch: amd64; os.version: 3.13.0-55-generic

20:19:02.024 [org.n52.sos.importer.feeder.task.OneTimeFeeder] 
    INFO  OneTimeFeeder - Starting feeding data from file via configuration 
    'sensordata2.config.xml' to SOS instance
20:19:02.039 [org.n52.sos.importer.feeder.task.OneTimeFeeder] 
    TRACE SensorObservationService - SensorObservationService(Configuration 
    [file=/home/john/Desktop/sensordata.config.xml])
20:19:02.196 [org.n52.sos.importer.feeder.task.OneTimeFeeder] 
    DEBUG SimpleHttpClient - executing POST method to 
    'http://localhost:8080/52n-sos-webapp/service'.
20:19:02.901 [org.n52.sos.importer.feeder.task.OneTimeFeeder] 
    INFO  SensorObservationService - Using 25ms timeout buffer during insert 
    observation requests. Change <SosImportConfiguration><SosMetadata 
    insertSweArrayObservationTimeoutBuffer> if required.
20:19:02.903 [org.n52.sos.importer.feeder.task.OneTimeFeeder] 
    DEBUG SensorObservationService - OperationsMetadata found: 
    org.n52.oxf.ows.capabilities.OperationsMetadata@3e0d0dba
20:19:02.918 [org.n52.sos.importer.feeder.task.OneTimeFeeder] 
    ERROR OneTimeFeeder - SOS 'http://localhost:8080/52n-sos-webapp/service' 
    does not support required transactional operations!
```

**Comment by EHJ-52n** [comment]
Did you activate the according operations, too? Configuring transactional security settings correct does not enable the transactional operations when using a newly installed instance of the 52N SOS.

**Comment by johnjohndoe** [comment]
Can you please point me to the settings which I have to configure? Screenshots are welcome.

**Comment by EHJ-52n** [comment]
```http://localhost:8080/52n-sos-webapp/admin/operations``` => ensure that the required transactional operations are enabled:

- InsertSensor
- InsertObservation

**Comment by johnjohndoe** [comment]
@EHJ-52n Great. Now the import executed without errors.
Please consider updating the error message so one knows that insert operations are disabled.

**Comment by nuest** [comment]
@johnjohndoe Would you mind creating an issue and tell the SOS developers what error message text you would expect (or what text would have helped you)? https://github.com/52North/sos/issues

Thanks.

**Comment by johnjohndoe** [comment]
@nuest The error message is output by the *sos-importer-feeder*. Therefore, I would suggest extending the original error message as such:

> ERROR OneTimeFeeder - SOS 'http://localhost:8080/52n-sos-webapp/service' does not support required transactional operations! In order to import data you also must enable  transactional operation (InsertSensor, InsertObservation) on your SOS instance.

**Comment by EHJ-52n** [comment]
@johnjohndoe Thanks for the hint. Fixed by #45.

**Comment by johnjohndoe** [comment]
:+1:
